data/testdata/all_types.golden.json

```diff
@@ -0,0 +1,119 @@
+      { "name": "uint8_values", "type": "number" },
+      { "name": "nullable_uint8_values", "type": "number" },
+      { "name": "uint16_values", "type": "number" },
+      { "name": "nullable_uint16_values", "type": "number" },
+      { "name": "uint32_values", "type": "number" },
+      { "name": "nullable_uint32_values", "type": "number" },
+      { "name": "uint64_values", "type": "number" },
+      { "name": "nullable_uint64_values", "type": "number" },
+      { "name": "float32_values", "type": "number" },
+      { "name": "nullable_float32_values", "type": "number" },
+      { "name": "float64_values", "type": "number" },
+      { "name": "float64_nans", "type": "number" },
+      { "name": "nullable_float64_values", "type": "number" },
+      { "name": "bool_values", "type": "bool" },
+      { "name": "nullable_bool_values", "type": "bool" },
+      { "name": "timestamps", "type": "time" },
+      { "name": "timestamps", "type": "time" },
+      { "name": "nullable_timestamps", "type": "time" }
+    ]
+  },
+  "data": {
+    "fields": [
+      ["Go Min", "JS Min (for >= 64)", "0 / nil / misc", "JS Max (for >= 64)", "Go Max"],
+      ["Grafana", "❤️", null, "🦥", "update your unicode/font if no sloth, is 2019."],
+      [-128, -128, 0, 127, 127],
+      [-128, -128, null, 127, 127],
+      [-32768, -32768, 0, 32767, 32767],
+      [-32768, -32768, null, 32767, 32767],
+      [-2147483648, -2147483648, 1, 2147483647, 2147483647],
+      [-2147483648, -2147483648, null, 2147483647, 2147483647],
+      [-9223372036854775808, -9007199254740991, 1, 9007199254740991, 9223372036854775807],
+      [-9223372036854775808, -9007199254740991, null, 9007199254740991, 9223372036854775807],
+      [0, 0, 1, 255, 255],
+      [0, 0, null, 255, 255],
+      [0, 0, 1, 65535, 65535],
+      [0, 0, null, 65535, 65535],
+      [0, 0, 1, 4294967295, 4294967295],
+      [0, 0, null, 4294967295, 4294967295],
+      [0, 0, 1, 9007199254740991, 18446744073709551615],
+      [0, 0, null, 9007199254740991, 18446744073709551615],
+      [1e-45, 1e-45, 1, 3.4028235e38, 3.4028235e38],
+      [1e-45, 1e-45, null, 3.4028235e38, 3.4028235e38],
+      [5e-324, -9007199254740991, 1, 9007199254740991, 1.7976931348623157e308],
+      [null, null, 0, null, null],
+      [5e-324, -9007199254740991, null, 1.7976931348623157e308, 9007199254740991],
+      [true, false, true, true, false],
+      [true, false, null, true, false],
+      [0, 1568039445000, 1568039450000, 9007199254, 9223372036854],
+      [0, 1568039445000, 1568039450000, 9007199254, 9223372036854],
+      [0, 1568039445000, null, 9007199254, 9223372036854]
+    ],
+    "entities": [
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      { "NaN": [1, 3], "Inf": [4], "NegInf": [0] },
```

Comment:
since json can not handle these... we encode them explicitly.  Fortunately json-iterator has a nice exception when it hits one of these values